packages/picasso.js/src/core/component/tween.js

```diff
@@ -58,26 +58,26 @@ function tween({ old, current }, { renderer }, config) {
           exited.ips.push(interpolateObject(ids[key], extend({}, ids[key], { r: 0.0001, opacity: 0 })));
         }
       });
-      if (exited.ips.length) {
+      if (exited.ips.length >= 0) {
```

Comment:
Is it necessary to check like this? Why if so?

Reply:
yeah, it should be written better.